app/controllers/games_controller.rb

```diff
@@ -1,5 +1,18 @@
 class GamesController < ApplicationController 
 	def show 
 		render template: "games/#{params[:game]}"
-	end
+	end 
+
+	def new 
+		@game = Game.new 
+	end 
+
+	def create 
+		@game = Game.create 
+	end 
+
+	def update 
+		@game = Game.find(params[:id]) 
+		current_player.update_attribute(:game_id, @game.id) 
```

Comment:
I am not sure you have access to the current_player at this point. Based on the database logic, what I would do here is create a player object belonging to the current_user and current game. (Like the comments in the nomster app).